lib/encoder.py

```diff
@@ -766,7 +770,11 @@ def _FilesToEncodings(self, files, videofile, bitrate=None,
         encoder or self._FileNameToEncoder(full_filename),
         bitrate or _FileNameToBitrate(full_filename),
         videofile or _FileNameToVideofile(full_filename))
-      candidate.Recover()
+      try:
+        candidate.Recover()
+      except Error:
+        print 'Ignoring error in file %s' % full_filename
```

Comment:
Here you should print the content of the caught exception. Goes somewhat like this:
except Error as e:
  print 'Ignoring error %s inf file %s' % (str(e), full_filename)